pages/common/bosh.md

```diff
@@ -0,0 +1,35 @@
+# bosh
+
+> Cli tool to deploy and manage the bosh director.
+
+- Deploy the director:
+
+`bosh create-env {{bosh.yml}} --state={{state.json}} --vars-store={{creds.yml}} -v {{custparam}}`
```

Comment:
Do we generally need to use "custparam" ? We usually start with simple invocations of a command. And this already looks like a complicated one. So, it would be nice if the custparam can be removed.

Reply:
Unfortunately custom parameters when invoking bosh create-env are pretty standard, they are used for setting ip ranges etc.
example: 
http://mariash.github.io/learn-bosh/#setup_bosh_environment

now that i think about it, perhaps including create-env is not the way to go.
one usually only does this once, i think ill just remove create-env all together and focus on the commonly u sed stuff (which is what tldr is for if i understand correctly;-) )